Traducir.Web/src/App/Components/SuggestionHistoryFilters.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { NonUndefinedReactNode } from "../NonUndefinedReactNode";
+
+export interface ISuggestionHistoryFiltersProps {
+    userid: number;
+}
+
+export default class Filters extends React.Component<ISuggestionHistoryFiltersProps> {
+    public render(): NonUndefinedReactNode {
+        return <div className="row text-center mt-3">
+            <div className="col d-none d-lg-block">
+                <div className="btn-group" role="group" aria-label="Basic example">
+                    <Link to={"/suggestions/" + this.props.userid} className="btn btn-info">All</Link>
+                    <Link to={"/suggestions/" + this.props.userid + "?stateId=1"} className="btn btn-info">Created</Link>
```

Comment:
We could avoid having this `int` here (it's not evident what 1 is in this context) and use the `StringSuggestionState` (that's inside `SOStringSuggestion.ts`)

Reply:
this could be 

```
<Link to={`/suggestions/${this.props.userid}?state=${StringSuggestionState.Created}`} className="btn btn-info">Created</Link>
```